developing-with-nats/connecting/name.md

```diff
@@ -2,4 +2,88 @@
 
 Connections can be assigned a name which will appear in some of the server monitoring data. This name is not required but can help in debugging and testing.
```

Comment:
This document will require a listing in the outline in order to be displayed. Please provide outline heading and location in summary.md.

Reply:
done